README.rst

```diff
@@ -66,6 +66,50 @@ For JBIG2 compression:
 * `jbig2enc <https://github.com/agl/jbig2enc>`_ for JBIG2 compression (and PyMuPDF 1.19.0 or higher)
 
 
+Installation
+============
+
+First install dependencies. For example, in Ubuntu::
+
+
```

Comment:
Any reason you are always leaving 2 lines empty? I think 1 should be quite sufficient...

Reply:
I can fix that up

Reply:
> Any reason

When in Rome, do as the Romans do. :slightly_smiling_face: I looked at the first existing code block I found (under "Usage")